tests/integration/defs/accuracy/test_llm_api_pytorch.py

```diff
@@ -2845,3 +2845,115 @@ def test_auto_dtype(self):
                  kv_cache_config=self.kv_cache_config) as llm:
             task = MMMU(self.MODEL_NAME)
             task.evaluate(llm, sampling_params=self.sampling_params)
+
+
+class TestBeamSearch(LlmapiAccuracyTestHarness):
+    MODEL_NAME = "meta-llama/Llama-3.1-8B-Instruct"
+    MODEL_PATH = f"{llm_models_root()}/llama-3.1-model/Llama-3.1-8B-Instruct"
+
+    kv_cache_config = KvCacheConfig(max_tokens=10000)
+
+    @parametrize_with_ids("disable_overlap_scheduler", [False, True])
```

Comment:
These test cases can be merged into two test functions
1. test_auto_dtype_beam_search
2. test_fp8_beam_search